### DISCUSSION TITLE
Monitor Interface Drops?

### DISCUSSION THREAD

**Discussion by sjnhawk** [Security-Onion-Solutions/securityonion#10100] [Unsupported Versions]
This might be a newb question, but I was hoping that someone could shed some light on "Monitor Interface Drops" in Grafana. I recently had to change out a bad NIC on my standalone setup and am wondering if I took a hit on performance. 

This is a fresh install of 2.3.220 and is running on a small network of around 30 devices. Unfortunately I did not record my previous stats, so I'm not sure if there is much of a delta there.

I'm averaging roughly 27-32 mp/s on Monitor Interface Drops, and am showing 0% packet loss for Zeek, Suricata, and Stenographer. Is this expected/within normal thresholds? Or is something not right?

I ran so-test and received the following results.

![image](https://user-images.githubusercontent.com/67609108/230794433-88ec7fb4-458c-4f2c-a14f-fc516f3e14f0.png)

**Comment by InfosecGoon** [comment]
Sometimes Monitor Interface Drops are a symptom of an overwhelmed or misconfigured SPAN port or tap -- how are you getting data into your standalone?

**Reply by sjnhawk** [reply]
I've got a SPAN port configured on a Unifi switch connected to my pfSense Firewall.
